README.md

```diff
@@ -1,3 +1,45 @@
-# language-cylc package for Atom
+# Cylc language package for Atom
 
 An Atom extension that provides language support for Cylc workflow configuration files.
+
+## Features
+
+Syntax highlighting:
+- Cylc 7
+- Cylc 8
+- ISO 8601 datetimes
+- Jinja2 (can be used with a Jinja extension e.g. [atom-jinja2](https://atom.io/packages/atom-jinja2))
+
+Check [here](https://github.com/cylc/cylc-flow/issues/2752) for info on supported syntax features.
+
+## Installation
+
+Currently, the only way to install is to clone the repository into your `.atom/` directory:
```

Comment:
Is this true if you are pushing it to the atom package repo?

Reply:
Just updated the instructions